jenkins/build-pod.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: v1
+kind: Pod
+spec:
+  containers:
+  - name: py27
+    image: circleci/python:2.7
+    command:
+    - cat
+    tty: true
+    securityContext:
```

Comment:
try to avoid this: securityContext:
      runAsUser: 0
      privileged: true

this mean you running as root and it's security risk if someone get access to this container.
use only in case you willy need it.